File: java/Factorial.java
public class Algorithms {  
    static int factorial(int n) { 
        if(n == 0) 
           return 1; 
        int i = n, fact = 1; 
        while (n / i != n) { 
            fact = fact * i; 
            i--; 
        } 
        return fact; 
    } 
  
    public static void main(String[] args) { 
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();
        System.out.println("Factorial of "+num+" is "+factorial(5)); 
    } 
} 

File: python/Factorial.py
def factorial(num): 
    if(num == 0): 
       return 1
       
    i = num 
    fact = 1
      
    while(num / i != num): 
        fact = fact * i 
        i -= 1
          
    return fact 
  
num = 5
print("Factorial of", num, "is",factorial(num)) 
   